src/verifier/step4/SumcheckLogic.sol

```diff
@@ -0,0 +1,488 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "src/pasta/Pallas.sol";
+import "src/pasta/Vesta.sol";
+import "src/verifier/step4/KeccakTranscript.sol";
+import "src/verifier/step4/EqPolynomial.sol";
+
+library PallasPolyLib {
+    struct MLPoly {
```

Comment:
As far as I see, these structs (MLPoly, SparseEntry, SparsePoly, UniPoly, CompressedUniPoly) are identical for both PallasPolyLib and VestaPolyLib. Can't we have them defined in some single place?